documents/motions/07.2020.1 - Registered Speedcubers.md

```diff
@@ -24,5 +24,5 @@ Persons who are Registered Speedcubers have the role of Non-voting Member of the
    1. Registered Speedcubers must behave according to the Spirit of the WCA within the WCA Community.
    2. Registered Speedcubers must comply with the Regulations before, during, and after a WCA Competition that they register for and/or participate in.
 7. Revocation of registration as a Registered Speedcuber
-   1. It is not possible to remove the registration of a Registered Speedcuber in the WCA.
-   2. Acknowledged WCA Competition results and Registered Speedcuber profiles should not be removed.
+   1. The registration of a Registered Speedcuber may be resigned from the WCA by notification in writing to the Secretary.
```

Comment:
What does resignation from the WCA as a registered speedcuber mean? Is this the same as the anonymization process outlined in the next point (if so, should this be contact WDPC instead of the Secretary)?

Reply:
This is a change of the Motions to match the Bylaws (Specifically 2.4 resignation and termination) As we don't specifically use memberships or membership categories at the moment this doesn't really change how anything is done, just makes our documents match

Reply:
I do not like that this is going to be removed:
2. Acknowledged WCA Competition results and Registered Speedcuber profiles should not be removed.

The next point does not cover this. I think we should not remove results. It is like changing the past and has bad effects (for example no winner of a competition),